package.json

```diff
@@ -22,8 +22,13 @@
     "start": "dotenv -- turbo run start --cache-dir=.turbo",
     "typecheck": "turbo run typecheck --cache-dir=.turbo"
   },
+  "dependencies": {
```

Comment:
These dependencies are probably accidentally added to root and should be added to the web and relevant packages instead